tests/integration/test_authd/test_authd_use_password.py

```diff
@@ -0,0 +1,224 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: This module verifies the correct behavior of the setting 'use_password'.
+
+components:
+    - authd
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-authd
+    - wazuh-db
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+tags:
+    - enrollment
+'''
+import os
+import time
+import pytest
+from pathlib import Path
+import re
+
+from wazuh_testing.constants.paths.configurations import DEFAULT_AUTHD_PASS_PATH
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.utils.configuration import load_configuration_template, get_test_cases_data
+
+from . import CONFIGURATIONS_FOLDER_PATH, TEST_CASES_FOLDER_PATH
+
+# Marks
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+
+# Configurations
+test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'config_authd_use_password.yaml')
+test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'cases_authd_use_password.yaml')
+test_configuration, test_metadata, test_cases_ids = get_test_cases_data(test_cases_path)
+test_configuration = load_configuration_template(test_configuration_path, test_configuration, test_metadata)
+
+# Variables
+DEFAULT_TEST_PASSWORD = 'TopSecret'
+AGENT_INPUT = "OSSEC A:'{}'"
+AGENT_INPUT_WITH_PASS = "OSSEC PASS: {} OSSEC A:'{}'"
+INVALID_REQUEST_MESSAGE = 'ERROR: Invalid request for new agent'
+INVALID_PASSWORD_MESSAGE = 'ERROR: Invalid password'
+SUCCESS_MESSAGE = "OSSEC K:'001 {} any "
+
+receiver_sockets_params = [(("localhost", 1515), 'AF_INET', 'SSL_TLSv1_2')]
+monitored_sockets_params = [('wazuh-modulesd', None, True), ('wazuh-db', None, True), ('wazuh-authd', None, True)]
+receiver_sockets, monitored_sockets = None, None
+
+
+# Functions
+
+def read_random_pass():
+    """
+    Search for the random password creation in Wazuh logs
+    """
+    passw = None
+    try:
+        with open(WAZUH_LOG_PATH, 'r') as log_file:
+            lines = log_file.readlines()
+            for line in lines:
+                if "Random password" in line:
+                    passw = line.split()[-1]
+            log_file.close()
+    except IOError as exception:
+        raise
+    return passw
+
+# Fixtures
+
+
+@pytest.fixture(scope='function')
```

Comment:
```suggestion
@pytest.fixture()
```
Also, move to `conftest.py` file from auth tests.